Sources/TerminalAPIKit/Models/Operation.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Operation.swift
+//  TerminalAPIKit
+//
+//  Created by Andrew Gates on 8/8/23.
+//
+
+import Foundation
+
+public final class Operation: Codable {
+
+    /// the type of request after which the waiting screen shows: InputRequest, DisplayRequest, and Payment
+    public let type:OperationType
```

Comment:
can we have a space between the variable name and the type? (applies to multiple places in the PR)